modules-js/react-fleet/src/components/DetailsDisclosure.tsx

```diff
@@ -1,15 +1,25 @@
 /** @jsx jsx */
 
-import { useEffect, useState, ReactChild } from 'react';
-
 import { css, jsx } from '@emotion/core';
 
-import {
-  CLEAR_DEFAULT_STYLING,
-  OPTIMISTIC_BLUE_DARK,
-} from '@cityofboston/react-fleet';
+import { useEffect, useState, Fragment, ReactChild } from 'react';
+
+import { CLEAR_DEFAULT_STYLING } from '../utilities/css';
+import { OPTIMISTIC_BLUE_DARK } from '../utilities/constants';
 
-export default function ApostilleRequestInstructions(): JSX.Element {
+interface Props {
+  summaryContent: string;
```

Comment:
Should make this `ReactChild` so one could put `<b>` or something in the summary.